hls4ml/converters/keras/recurrent.py

```diff
@@ -47,4 +47,6 @@ def parse_rnn_layer(keras_layer, input_names, input_shapes, data_reader):
     if layer['return_state']:
         raise Exception('"return_state" of {} layer is not yet supported.')
 
+    layer['pass_initial_states'] = False
```

Comment:
Not quite, right? We find out from number of inputs. Also, would be good to add this as an expected attribute to the RNN layers in IR